src/swagger2.0.json

```diff
@@ -641,6 +692,262 @@
                 }
             }
         },
+        "Geometry": {
```

Comment:
Gedachtes...

Ik dacht even aan het includen van deze meuk via een externe link `$ref: './includes/geo.yaml#/components/Geometry'` maar in retrospect lijkt me dat toch niet wenselijk ivm linting, parsing en... alles bij elkaar :)

Het kan echter wel, met het advies dat ook de includes geldige OAS specs moeten zijn: https://dev.to/mikeralphson/defining-reusable-components-with-the-openapi-specification-4077

Reply:
Zou het inderdaad wat opruimen, maar ik weet niet in hoeverre de tooling hiervoor klaar is, en dan moet ik drf-yasg ook wijsmaken dat stukken in een ander bestand terechtkomen. Ik houd het in het achterhoofd :)

Reply:
Ja, zeker ook niet iets om nu door te voeren.